.yamllint

```diff
@@ -18,5 +18,6 @@ rules:
 ignore: |
   .tox
   .git
+  .github
```

Comment:
Why don't we want to yamllint or ansible-lint these files?

Reply:
yamllint, i am not sure. Maybe gha wants the yaml in a different flavor.
ansible-lint, however looks for all yaml files and tries to interpret them as playbooks; not particularly helpful.

Reply:
I had some weird issue locally with github actions yaml (maybe it is on my pyenv).
I can remove this change

Reply:
It sounds like .ansible-lint should ignore .github/workflows/ rather than all of .github . That should be a separate change though.